pgmpy/tests/test_models/test_BayesianModel.py

```diff
@@ -118,6 +118,11 @@ def test_moral_graph_with_edge_present_over_parents(self):
             self.assertTrue(edge in [('a', 'b'), ('c', 'b'), ('d', 'a'), ('d', 'b'), ('d', 'e')] or
                             (edge[1], edge[0]) in [('a', 'b'), ('c', 'b'), ('d', 'a'), ('d', 'b'), ('d', 'e')])
 
+    
+    def test_local_independencies(self):
+        from pgmpy.independencies import Independencies
+        self.assertEqual(self.G.local_independencies('a'),Independencies(['a',['b','c']]))
```

Comment:
@ankurankan I think adding an ``__eq__`` method in the ``Independencies`` class would be a better choice, since it can be used in the future.